plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -49,6 +51,7 @@ export type CreateWorkerOptions = {
   workingDirectory: string;
   logger: Logger;
   additionalTemplateFilters?: Record<string, TemplateFilter>;
+  concurrentTasksLimit?: number;
```

Comment:
Some docs stating the default 10 and that `Infinity` is the thing you should set if you don't want any limit

Reply:
Thanks @jhaals, which file should this docs content be added? Or do you mean inline as JSDocs?

Reply:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/338265/196295202-15e76430-20a2-4256-9ec1-c223445be9bd.png">